qiskit/qobj/pulse_qobj.py

```diff
@@ -649,7 +649,7 @@ def to_dict(self):
         for use with IBMQ you can leverage a json encoder that converts these
         as expected. For example:
 
-        .. code-block::
+        .. code-block:: python
```

Comment:
How come this one stayed as-is?

(Fwiw, `python` should be the default language for the Qiskit build, so I think it ought to have been highlighting correctly anyway.)

Reply:
I think I wasn't sure how to make a pulse qobj and decided to leave it since this class is deprecated.